packages/jds/src/components/Dialog/Dialog.styles.ts

```diff
@@ -0,0 +1,128 @@
+import { keyframes } from "@emotion/react";
+import styled from "@emotion/styled";
+import * as DialogPrimitive from "@radix-ui/react-dialog";
+import { pxToRem, shadow } from "utils";
+
+import { BlockButton, Title } from "@/components";
+
+const dialogOverlayFadeIn = keyframes`
+  from { opacity: 0 }
+  to { opacity: 1 }
+`;
+
+const dialogOverlayFadeOut = keyframes`
+  from { opacity: 1 }
+  to { opacity: 0 }
+`;
```

Comment:
(참고) 이런 부분들이 반복되어서 사용된다면


```suggestion
const 애니메이션공통함수 = (from: number, to: number) => keyframes`
  from { opacity: ${from} }
  to { opacity: ${to} }
`;
```

요렇게 처리가 되어도 좋을 거 같네요~